parlai/core/agents.py

```diff
@@ -375,21 +375,39 @@ def load_agent_module(opt):
     model_file = opt['model_file']
     optfile = model_file + '.opt'
     if os.path.isfile(optfile):
+        # Note: new_opt is a bit of a misnomer: opt is what comes from the CLI args,
+        # and new_opt is the options on disk
```

Comment:
"options on disk" is a bit confusing, maybe "options loaded from the opt file associated with the model file that is being loaded" ... that might be more confusing